src/clients/FtpRelay.cc

```diff
@@ -784,12 +786,9 @@ Ftp::Relay::stopOriginWait(int code)
 }
 
 void
-Ftp::Relay::abort(void *d)
+Ftp::Relay::Abort(Relay *ftpClient)
```

Comment:
```suggestion
Ftp::Relay::HandleStoreAbort(Relay *ftpClient)
```

Same for all other abort handlers we modify in this PR. This is a Store abort handler rather than a method that has to abort the given job.